Airbrake_Ideas.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
```

Comment:
I would also import cylloger (in helper objects) to log certain events in this main file